build.Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN mkdir -p /usr/local/bin \
 
 # Download docker
 RUN mkdir -p /usr/local/bin \
-    && curl -fL https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}-ce.tgz \
+    && curl -fL https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz \
```

Comment:
When I built the `traefik-dev` image from build.Dockerfile, I noticed the `DOCKER_VERSION` environment variables was set to `17.09.0-ce`. Maybe Docker changed this recently? Anyway, the build wouldn't complete because it kept trying to download `docker-17.09.0-ce-ce.tgz`, which doesn't work so well.